src/sample.ts

```diff
@@ -1,19 +1,20 @@
-const SimpleKeyring = require('.');
+import SimpleKeyring from '.';
```

Comment:
It still serves as a nice executable sample, and I appreciated it myself when I was converting to TS.

How about if we add it to the `exclude` section of `tsconfig.build.json`?

Reply:
Sure. I like to keep `src/` clean of non-shippable files, but we don't currently do that anyway with test files, so that's not really a rule. Seems fine to me to exclude it in that case.